spec/honeycomb/client_spec.rb

```diff
@@ -155,4 +155,18 @@
 
     it_behaves_like "event data", package_fields: false
   end
+
+  describe "sending from within a span block" do
+    it "does not also send the parent span" do
+      root_span = client.start_span(name: "root")
+      client.start_span(name: "child", &:send)
+      expect(root_span.__send__(:sent?)).to be false
```

Comment:
Is there a specific reason to use `__send__` that I'm not seeing? If not lets check the events that libhoney is told to send rather than use the `__send__` here.

something like:

`expect(libhoney_client.events.size).to eq 1`

Reply:
Good call. I had that thought about the private method call last night, then forgot about it 😳

I just pushed a new commit with your suggestion. My other thought was `expect(client.current_span).to eq root_span`, but I think they are effectively checking the same thing.

I also reformatted the `&:send` back to block form and disabled that cop for those blocks.